src/main/scala/uk/gov/hmrc/play/http/HttpGet.scala

```diff
@@ -34,4 +34,19 @@ trait HttpGet extends HttpVerb with ConnectionTracing with HttpHooks {
     executeHooks(url, GET_VERB, None, httpResponse)
     mapErrors(GET_VERB, url, httpResponse).map(response => rds.read(GET_VERB, url, response))
   }
+
+  def GET[A](url: String, queryParams: Seq[(String, String)])(implicit rds: HttpReads[A], hc: HeaderCarrier): Future[A] =withTracing(GET_VERB, url) {
```

Comment:
I just thought the logic would be cleaner this way, rather than having to check if the url contains "?" but only if the user specified no queryParams. To the end user it makes little difference.